pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -189,7 +189,8 @@ class CreateVM extends React.Component {
             installationSource = (
                 <FileAutoComplete id={installationSourceId}
                     placeholder={_("Path to ISO file on host's file system")}
-                    onChange={this.onChangedValue.bind(this, 'source')} />
+                    onChange={this.onChangedValue.bind(this, 'source')}
+                    superUser="try"/>
```

Comment:
This is to see files in user-inaccessible paths? Makes sense. (Please add the justification to the commit message)

Reply:
Done